tools/ogstools/Extract/.shed.yml

```diff
@@ -0,0 +1,18 @@
+name: suite_ogs_tools
+owner: ufz
+description: A suite of tools for OpenGeoSys (ogs)
+long_description: |
+     A suite of tools for ogs.
+categories:
+- Ecology
+remote_repository_url: https://gitlab.opengeosys.org/ogs/tools/ogstools
```

Comment:
This should be `https://github.com/Helmholtz-UFZ/galaxy-tools/tree/main/tools/ogs/`